src/components/TimePicker/pageObject/index.js

```diff
@@ -0,0 +1,237 @@
+const PageModal = require('../../Modal/pageObject');
+
+/**
+ * TimePicker page object class.
+ * @class
+ * @tutorial picklist
+ */
+class PageTimePicker {
+    /**
+     * Create a new TimePicker page object.
+     * @constructor
+     * @param {string} rootElement - The selector of the TimePicker root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+        this.timePickerModal = '#time-picker_modal';
+    }
+
+    /**
+     * Clicks the input element.
+     * @method
+     */
+    click() {
+        $(this.rootElement)
+            .$('input')
```

Comment:
since there are other inputs within this component you need to be more specific in this selector